### PR TITLE
Log in (UI)

### PR DESCRIPTION
**Pull request by Ernest103**
Changes to main.dart, Some progress made to log in screens

### CODE REVIEW

File: tip_tracker/lib/main.dart [hunk 1]
```diff
@@ -10,106 +11,10 @@ class MyApp extends StatelessWidget {
...
-        // Notice that the counter didn't reset back to zero; the application
-        // is not restarted.
-        primarySwatch: Colors.blue,
-      ),
-      home: const MyHomePage(title: 'Flutter Demo Home Page'),
-    );
-  }
-}
-
-class MyHomePage extends StatefulWidget {
```
Comment (Samuelmm97) [inline tip_tracker/lib/main.dart:32]: Was all this unnecessary code before?

File: tip_tracker/pubspec.yaml [hunk 2]
```diff
@@ -60,8 +60,8 @@ flutter:
   uses-material-design: true
 
   # To add assets to your application, add an assets section, like this:
-  assets:
```
Comment (Samuelmm97) [inline tip_tracker/pubspec.yaml:63]: test comment

**Comment by MVeneroY** [comment]
test

File: tip_tracker/lib/main.dart (hunk 1 above)
Comment (Ernest103) [reply tip_tracker/lib/main.dart:32]: All of these was from the pre-generated sample app which flutter creates

**Review by Samuelmm97 — APPROVED** [review]
LGTM, I'd wait for @brandongevat to review though

**Pull request closed without merging**